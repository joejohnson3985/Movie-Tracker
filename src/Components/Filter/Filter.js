import React, {Component} from 'react';

class Filter extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="search-bar">
        <select>
          <option value="action">Genre</option>
          <option value="action">Action</option>
          <option value="adventure">Adventure</option>
          <option value="comedy">Comedy</option>
          <option value="horror">Horror</option>
        </select>
      </div>
    );
  }
}

export default Filter;
