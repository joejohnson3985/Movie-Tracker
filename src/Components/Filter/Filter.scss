@import "../../Utilities/_Variables.scss";

.drop-down {
  display: flex;
  align-items: center;
  justify-content: center;
  text-decoration: none;
  color: #595959;
  font-size: 22px;
  background-color: #d38787;
  margin: 10px auto;
}

.search {
  display: flex;
  align-items: center;
}

.search-input {
  width: 400px;
  height: 30px;
}

.search-button {
  height: 35px;
  width: 70px;
  background-color: #d38787;
  color: $main-text;
}

.filter-bar {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 20px auto;
}
