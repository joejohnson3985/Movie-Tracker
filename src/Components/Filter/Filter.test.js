import React from 'react'
import { shallow, configure } from 'enzyme';
import  Filter  from './Filter'
import Adapter from 'enzyme-adapter-react-16';


describe('Filter', () => {
  it('should match the snapshot', () =>{
    const wrapper = shallow(<Filter/>)
    expect(wrapper).toMatchSnapshot()
  })
 
})



