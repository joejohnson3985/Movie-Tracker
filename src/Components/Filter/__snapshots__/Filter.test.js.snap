// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter should match the snapshot 1`] = `ShallowWrapper {}`;
